CONTRIBUTING.md

```diff
@@ -0,0 +1,36 @@
+# Contributing to Aloxide
+
+1. [Getting Involved](#getting-involved)
+2. [How To Develop Features](#how-to-develop)
```

Comment:
Add, create, or suggest etc. is more specific. Develop is a generic word.

Reply:
I split it into smaller detailed sections in `How To Develop Features` so I think we don't need to make it so clear here.